test/contrasts.jl

```diff
@@ -42,7 +44,7 @@ end
                             1  0  0
                             1  0  0
                             1  1  0]
-@test coefnames(mf) == ["(Intercept)"; "x: b"; "x: c"]
+@test StatsBase.coefnames(mf) == ["(Intercept)"; "x: b"; "x: c"]
```

Comment:
No need to fully qualify exported functions when calling them (only when overloading them).

Reply:
`coefnames` is not exported in the latest tagged StatsBase (it is in master, but not in the latest release which is the one Travis uses). As soon as the next StatsBase version is tagged then we can drop the prefix.

Reply:
Then using the fully qualified name hides a breaking change: `coefnames` used to be exported by `StatsModels`, and now it's no longer available. Unfortunately, I'm not sure there's a good way to handle this: deprecating `coefnames` here but still exporting it creates a name clash with StatsBase even if that's the same function. I guess we could remove `coefnames` without prior notice, knowing that StatsModels is very young and everything is in transition yet.

At any rate, better require StatsBase 0.19.2: https://github.com/JuliaLang/METADATA.jl/pull/12279

Reply:
Thanks for the tag. Since StatsModels is meant to be used with `StatsBase` either with `StatsModels.DataFrameStatisticalModel` or `StatsBase.StatisticalModel` should we `@rexport` StatsBase and DataFrames?
It makes more sense for
```
using StatsModels
```
rather than
```
using DataFrames, StatsBase, StatsModels
```

Reply:
Maybe, but that's a difficult question that should be tackled separately.

I've merged the StatsBase 0.19.2 PR.